proton-c/CMakeLists.txt

```diff
@@ -729,6 +729,12 @@ if (BUILD_PYTHON)
 
 endif (BUILD_PYTHON)
 
+# fuzz tests: tests/fuzz
+option(FUZZ_TEST "Enable building fuzzers and regression testing with libFuzzer" OFF)
+if (FUZZ_TEST)
+  add_subdirectory(src/tests/fuzz)
+endif (FUZZ_TEST)
```

Comment:
done, added another cmake variable `FUZZING_ENGINE` which decides whether to link with libFuzzingEngine or with a simple driver. It defaults to the simple driver.